### ISSUE TITLE
Update threejs version

### ISSUE THREAD

**Issue by jubeira** [osrf/polymer-ros-rviz#14]
We are trying to upgrade the robot web tools dependencies to the latest ones in an effort to improve performace; see https://github.com/osrf/rvizweb/issues/8.

From https://github.com/osrf/rvizweb/issues/8#issuecomment-449963799:
- The current codebase uses `ros3djs` at a specific hash: https://github.com/osrf/polymer-ros-rviz/blob/37bd0fb92bfd776bfc59f215b910655eccf37991/bower.json#L39
which depends on `three.js#r64`.
- The latest `ros3djs` (`develop` branch) uses `three.js#r88`.

When upgrading both `ros3djs` and `threejs`, a `TypeError` is thrown when initializing the viewer:
```
TypeError: THREE.EventDispatcher.prototype.apply is not a function
```

This is coming from a file in the codebase (STLLoader), and the function is defined in here: https://github.com/osrf/polymer-ros-rviz/blob/0cb36be409dab31501f1d6de2d65a432471f3149/scripts/threejs/three.js#L6666-L6679.

My guess is that the new threejs required by ros3djs (r88) is redefining the event dispatcher somewhere. This issue is to track findings and progress regarding these upgrades.

**Comment by jubeira** [comment]
In https://github.com/mrdoob/three.js/releases/tag/r84, the changelog says the `apply` function has been removed from `EventDispatcher`.
Changing `STLLoader` and `ColladaLoader` with the files in `r84` seems to solve the problem.

The next issue is that `ros3djs` cannot find `ROSLIB` reference. I gave a shot at `master` branch instead of `develop` and the error was gone.

While doing this, I had changed the version of `roslibjs` in `ros-websocket` to match the ones being used by `ros3djs` here: https://github.com/jubeira/ros-websocket/commit/a9ce55859c78c292dfc642e69ffb3092c80c094d
This caused an issue with the reference to `ros-websocket`:
```
TypeError: websocket.connect is not a function
```
Apparently it's not necessary to make the versions match in this case. I rolled back the changes to the deps for `ros-websocket` and `ros-service` and I'm not having JS errors anymore.
Here's the branch with the WIP so far: https://github.com/osrf/polymer-ros-rviz/tree/jubeira/update_robot_web_tools.

With this I "could" load PR2 but it's not being properly displayed (no JS errors on the console though, just some logs):
![image](https://user-images.githubusercontent.com/2480899/50563363-fe574400-0cfa-11e9-8dd9-82340a8870b1.png)

**Comment by jubeira** [comment]
Update: I tried to use the pointcloud but I didn't get better results with `ros3djs @ master` with respect to the current `ros3djs` version. 

I went to `ros3djs` repository to try out the examples for the pointcloud, and I couldn't get it to work properly with neither `master` nor `develop` branches. Details here: https://github.com/RobotWebTools/ros3djs/issues/243.

We need `ros3djs` to work properly, as `ros-rviz` is just a wrapper over it. In short, at `master` branch I could see the pointcloud but with a huge delay (no GPU usage). At `develop` branch I hit a different error, so I opened the issue above to get some pointers.

The goal would be to have a version of `ros3djs` running using `webGL` with the graphics card, and then update `ros-rviz` component to use it.
